src/webgl/material.js

```diff
@@ -343,4 +323,34 @@ p5.RendererGL.prototype._applyColorBlend = function(v1,v2,v3,a){
   return colors;
 };
 
+
+p5.prototype.loadShader = function(fragShader, vertShader) {
+  var loadedShader = new p5.Shader();
```

Comment:
This gets into a break with the Processing PShader API that we should probably discuss: in Processing, the arguments for the constructor are the frag and vert file paths. In p5.Shader, I switched to using the actual frag and vert source code as creation arguments, which works better for default shaders (and in the case where, for example, someone wanted to make a p5 project that pulled shader source from somewhere else, like a DOM node). However, it means that loadShader has to return an empty shader object at first, and then asynchronously populate the sources of that shader, once those sources actually load.